app/Http/Controllers/AdminsController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use Hash;
+use Validator;
+
+class AdminsController extends Controller
+{
+
+    function NewAdmin (Request $r)
+    {
+    	$validation=Validator::make($r->all(),
+    	[
```

Comment:
fnc(arg1, arg2)
fnc(
  arg1,
  arg2
)

this is wrong style
fnc(arg1,
arg2)

Reply:
Look in the project that I've sent you for an example.